forest/oak/base.py

```diff
@@ -461,6 +462,10 @@ def run(study_folder: str, output_folder: str, tz_str: str = None,
         fs: integer
             sampling frequency
     """
+
+    # suppress warnings about mean of empty slices
+    warnings.simplefilter("ignore", category=RuntimeWarning)
```

Comment:
A better solution usually is to avoid getting a warning in the first place. Perhaps using `nanmean()`?
https://numpy.org/doc/stable/reference/generated/numpy.nanmean.html

Reply:
Both mean and nanmean return a warning when computed over empty slice, i.e., `RuntimeWarning: Mean of empty slice`.  I used a condition on `a` instead.

Reply:
OK, that works too. For future reference, I've just tested`nanmean()` using an example from the docs and it didn't raise a RuntimeWarning.
```python
>>> a = np.array([[1, np.nan], [3, 4]])
>>> np.nanmean(a)
2.6666666666666665
```